### ISSUE TITLE
Qt ImportError when running on GPU

### ISSUE THREAD

**Issue by Hugo-Trentesaux** [alicevision/meshroom#438]
When I launch meshroom without GPU, I get the interface and the pipeline running until mesh reconstruction where it complains not finding cuda enabled gpu:

```
[20:25:12.137663][warning] Could not determine number of CUDA cards in this system
[20:25:12.074072][warning] No CUDA-Enabled GPU.
[20:25:12.138309][error] cudaGetDeviceCount failed: no CUDA-capable device is detected
[20:25:12.138339][info] Can't find CUDA-Enabled GPU.
[20:25:12.138362][error] This program needs a CUDA-Enabled GPU (with at least compute capablility 2.0).
```

When I launch meshroom with bumblebee optirun or nvidia-xrun, it complains about not being able to import Qt (while other Qt program works fine):

```
ImportError: /usr/share/meshroom-bin/lib/PySide2/Qt/lib/libQt53DCore.so.5: symbol _ZdlPvm version Qt_5 not defined in file libQt5Core.so.5 with link time reference
```

I installed meshroom and nvidia-xrun from AUR, and cuda and bumblebee from official Arch repositories.

**Comment by richardjuan** [comment]
I get  the same error, using the official tar archive.

**Comment by ampotos** [comment]
I have the same issue, I'm trying to find a fix but nothing yet.

**Comment by ampotos** [comment]
For what I've seen it's potentially a mismatch between the QT5 version shipped with meshroom and the on on your system.  It looks like when not using bumblebee meshroom is using the library shipped with it but not when using optirun/nvidia-xrun.
I tried to play with LD_LIBRARY_PATH without any success.
It looks like it's only a qt issue which mean that it should be possible to use the CLI tool with bumleebe.

**Comment by fabiencastan** [comment]
This has been fixed in Meshroom 2019.2 with the upgrade of Qt version (which includes a fix regarding that).
